### PR TITLE
Add blog post about release of autoupgrade v6

### LINKED ISSUE #1301: [Article idea] Autoupgrade module v6.0.0 release
Hello!
We will shortly be publishing an article on the release of version 6.0.0 of the Autoupgrade module.
Thanks!

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | This PR adds the announcement of the latest release of autoupgrade module.
| Fixed ticket? | Fixes https://github.com/PrestaShop/prestashop.github.io/issues/1301

image

### CODE REVIEW

**Review by M0rgan01 — APPROVED** [review]

File: content/news/posts/2024/autoupgrade-v6-release.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+* **Improved data privacy -** Sanitize generated update logs from sensible data, such as the admin folder name and adding a link to updated privacy policy.
+* **Smoother update process -** Allow the whole update process to continue even when a module triggers an exception during the run of its migration files.
+* **Reliability of local updates -** When using a local archive, automatic detection of the PrestaShop version and check for PrestaShop matching in the ZIP and XML.
+
+
+#### For developers:
+
+* **Crowdin translation management -** Rework the management and distribution of translations of the first native & open-source module with Crowdin and PrestaShop module translation tool.
+* **Update translation format -** Switch to translations in XLIFF format.
+Storybook front preview - Introduce Storybook on the project to easy the incoming UI rework.
```
Comment (tblivet) [inline content/news/posts/2024/autoupgrade-v6-release.md]:
```suggestion
* **Storybook front preview -** Introduce Storybook on the project to easy the incoming UI rework.
```

File: content/news/posts/2024/autoupgrade-v6-release.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+* **Deploying automated tests -** Introduce UI tests running nightly.
+Improving code quality - Increase number of checks made by PHPStan.
```
Comment (tblivet) [inline content/news/posts/2024/autoupgrade-v6-release.md]:
```suggestion
* **Improving code quality -** Increase number of checks made by PHPStan.
```

File: content/news/posts/2024/autoupgrade-v6-release.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+{{< cta "https://github.com/PrestaShop/autoupgrade/releases/tag/v6.0.0" >}}Check out what’s new{{< /cta >}}
+
+### Download the new version
+
+You should be able to download or update the latest module version from your Back Office. You can also access the module ZIP file in the project's GitHub repository by clicking on the link below.
+
+{{< cta "https://github.com/PrestaShop/autoupgrade/releases/download/v6.0.0/autoupgrade-v6.0.0.zip" >}}Download Autoupgrade v6.0.0 now!{{< /cta >}}
+
+If you encounter any problems during the update that, in your opinion, are not caused by third-party modules or modifications used in your shop, submit a bug report.
```
Comment (tblivet) [inline content/news/posts/2024/autoupgrade-v6-release.md]:
```suggestion
If you encounter any problems during the update that, in your opinion, are not caused by third-party modules or modifications used in your shop, [submit a bug report](https://www.prestashop-project.org/get-involved/report-issues/).
```

**Review by kpodemski — CHANGES_REQUESTED** [review]
I'm blocking the PR because it has to be validated by the communication team.

File: content/news/posts/2024/autoupgrade-v6-release.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Autoupgrade module v6.0.0 is Available (1-Click Upgrade)"
```
Comment (coffrep) [inline content/news/posts/2024/autoupgrade-v6-release.md:3]:
```suggestion
title: "Autoupgrade Module v6.0.0 is Available (1-Click Upgrade)"
```

File: content/news/posts/2024/autoupgrade-v6-release.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+subtitle: "We are pleased to introduce the release of this new major version of the Autoupgrade module (also known as 1-Click Upgrade)."
```
Comment (coffrep) [inline content/news/posts/2024/autoupgrade-v6-release.md]:
```suggestion
subtitle: "We are pleased to introduce the new major version release of the Autoupgrade module, also known as 1-Click Upgrade."
```

Comment (coffrep) [reply content/news/posts/2024/autoupgrade-v6-release.md]: Took out the brackets for easier reading

File: content/news/posts/2024/autoupgrade-v6-release.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+date:   2024-07-29
+authors: [ GericFosse ]
+icon: icon-leaf
+tags: [ module, autoupgrade ]
+---
+
+In the footsteps of the latest module releases ([see versions 5.0.2 and 5.0.3 live presentation](https://build.prestashop-project.org/news/2024/live-update-june-2024/)), this version brings a number of improvements to streamline PrestaShop updates. Explore the changelog below to find out what version 6.0.0 of the Autoupgrade module has to offer.
```
Comment (coffrep) [inline content/news/posts/2024/autoupgrade-v6-release.md]:
```suggestion
In the footsteps of the latest module releases ([see versions 5.0.2 and 5.0.3 live presentation](https://build.prestashop-project.org/news/2024/live-update-june-2024/)), this version brings a number of improvements to streamline PrestaShop updates. Explore the changelog below to find out what version 6.0.0 brings to the Autoupgrade module.
```

File: content/news/posts/2024/autoupgrade-v6-release.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+![1-click upgrade module logo](/assets/images/2024/07/autoupgrade-logo.png)
+
+### What’s inside
+
+#### Global improvements:
+
+* **Ensure update path -** Optimization of the upgrade path to the latest version of PrestaShop 8 when using the minor channel.
+* **Enable usage & error analytics -** Introduce anonymous analytics with Segment and voluntary error reporting based on Sentry.
+* **Better display of RTL languages -** Fix Right To Left languages display in the BO by better RTL files management during updates.
```
Comment (coffrep) [inline content/news/posts/2024/autoupgrade-v6-release.md:21]:
```suggestion
* **Better display of RTL languages -** Fix Right To Left languages display in the BO by managing better RTL files during updates.
```

File: content/news/posts/2024/autoupgrade-v6-release.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+* **New PS/PHP compatibility check -** Adding a check to make sure the PHP version is compatible with the target PrestaShop before allowing the update to start.
+* **Improved data privacy -** Sanitize generated update logs from sensible data, such as the admin folder name and adding a link to updated privacy policy.
+* **Smoother update process -** Allow the whole update process to continue even when a module triggers an exception during the run of its migration files.
+* **Reliability of local updates -** When using a local archive, automatic detection of the PrestaShop version and check for PrestaShop matching in the ZIP and XML.
+
+
+#### For developers:
+
+* **Crowdin translation management -** Rework the management and distribution of translations of the first native & open-source module with Crowdin and PrestaShop module translation tool.
```
Comment (coffrep) [inline content/news/posts/2024/autoupgrade-v6-release.md]:
```suggestion
* **Crowdin translation management -** Rework the management and distribution of translations of the first native and open source module with Crowdin and PrestaShop module translation tool.
```

File: content/news/posts/2024/autoupgrade-v6-release.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+* **Update translation format -** Switch to translations in XLIFF format.
+* **Storybook front preview -** Introduce Storybook on the project to easy the incoming UI rework.
```
Comment (coffrep) [inline content/news/posts/2024/autoupgrade-v6-release.md]:
```suggestion
* **Storybook front preview -** Introduce Storybook on the project to ease the incoming UI rework.
```

File: content/news/posts/2024/autoupgrade-v6-release.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+* **Deploying automated tests -** Introduce UI tests running nightly.
+* **Improving code quality -** Increase number of checks made by PHPStan.
+
+
+The version v6.0.0 of the module also brings some new improvements, you can read the full Changelog by clicking on the link below.
```
Comment (coffrep) [inline content/news/posts/2024/autoupgrade-v6-release.md]:
```suggestion
The version 6.0.0 of the module also brings other new improvements, you can read the full changelog by clicking on the link below.
```

File: content/news/posts/2024/autoupgrade-v6-release.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+You should be able to download or update the latest module version from your Back Office. You can also access the module ZIP file in the project's GitHub repository by clicking on the link below.
+
+{{< cta "https://github.com/PrestaShop/autoupgrade/releases/download/v6.0.0/autoupgrade-v6.0.0.zip" >}}Download Autoupgrade v6.0.0 now!{{< /cta >}}
+
+If you encounter any problems during the update that, in your opinion, are not caused by third-party modules or modifications used in your shop, [submit a bug report](https://www.prestashop-project.org/get-involved/report-issues/).
+
+### Thank you, contributors
+
+The release of this new version of the module Autoupgrade is the result of a collective effort by all contributors to the project. We would therefore like to take this opportunity to thank all our contributors, from both the PrestaShop SA team and the community at large.
```
Comment (coffrep) [inline content/news/posts/2024/autoupgrade-v6-release.md]:
```suggestion
This new version release of the Autoupgrade module is the result of a collective effort by all project contributors. We are therefore taking this opportunity to thank all our contributors, from both the PrestaShop SA team and the community at large.
```

**Review by coffrep — APPROVED** [review]
Hi @kpodemski @gericfo very good article, I made a few edits and am approving for blog publishing, do let me know if you have questions.

File: content/news/posts/2024/autoupgrade-v6-release.md (hunk 7 above)
Comment (Quetzacoalt91) [reply content/news/posts/2024/autoupgrade-v6-release.md:21]: What about something like

```suggestion
* **Better display of RTL languages -** Fix Right To Left languages display in the BO with a systematic regeneration of RTL files during updates.
```

?

File: content/news/posts/2024/autoupgrade-v6-release.md (hunk 4 above)
Comment (Quetzacoalt91) [reply content/news/posts/2024/autoupgrade-v6-release.md:3]: What's the reason behind this suggestion? "Module" is not a part of the name but the regular extension as in the rest of the article.

Comment (kpodemski) [reply content/news/posts/2024/autoupgrade-v6-release.md:3]: @coffrep capitalization of the title is controlled via CSS, no need to capitalize it in the text

Comment (coffrep) [reply content/news/posts/2024/autoupgrade-v6-release.md:3]: Ok fine thanks for the info @kpodemski - yes in English @Quetzacoalt91 , titles have capitalized first letters in each word ;-)

File: content/news/posts/2024/autoupgrade-v6-release.md (hunk 7 above)
Comment (coffrep) [reply content/news/posts/2024/autoupgrade-v6-release.md:21]: Hi @Quetzacoalt91 sounds good to me. Good @kpodemski if you can use this version and not my version.

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thank you @Quetzacoalt91 the article will be live in a few minutes :)

**Comment by Quetzacoalt91** [comment]
Thank you @kpodemski & @coffrep :tada:

**Pull request merged**